module.js

```diff
@@ -2,12 +2,64 @@
 
 module.exports = {
 
+  extends: ['plugin:import/errors'],
+
+  plugins: ['import'],
+
   parserOptions: {
     sourceType: 'module'
   },
 
   rules: {
-    'func-name-matching': [2, {includeCommonJSModuleExports: true}]
-  }
+    'func-name-matching': [2, {includeCommonJSModuleExports: true}],
+
+    // Errors
+    'import/no-absolute-path': [2],
+    'import/no-cycle': [2],
+    'import/no-duplicates': [2],
+    'import/no-self-import': [2],
+
+    // Best practices
+    'import/export': [2],
+    'import/no-amd': [2],
+    'import/no-commonjs': [2],
+    'import/no-extraneous-dependencies': [2],
+    'import/no-import-module-exports': [2],
+    'import/no-mutable-exports': [2],
+    'import/no-named-as-default': [2],
+    'import/no-unused-modules': [2],
+    'import/no-useless-path-segments': [2],
+    'import/no-webpack-loader-syntax': [2],
+
+    // Style guide
+    'import/extensions': [2, 'ignorePackages'],
```

Comment:
Forcing use of the `.js` extension at the end of import statements makes our code more standards-compatible, possibly allowing it to run on Deno or natively in the browser. I figured it's a good practice to adopt.

Reply:
I think forcing a rule to potentially handle edge cases (Deno or native browser support) might be a bit too much, as we could just as well add the rule to any projects that would need it. Then again, I'm not strongly opinionated on this one